protos/farm_ng/core.proto

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) farm-ng, inc.
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+syntax = "proto3";
+
+package farm_ng.proto;
```

Comment:
What are pros/cons of ``farm_ng.proto`` versus ``farm_ng.core.proto`` ?  i think it might be wise to ensure that the package name for protobufs matches the project and path.

Reply:
Yeah, we can do farm_ng.core.protoand the sophus.proto. Basically matching the repo in the package name. The change should be easy to do if we decide to move forward.